docs/zkapps/advanced-snarkyjs/events.mdx

```diff
@@ -61,6 +61,34 @@ class SmartContract {
 }
 ```
 
+### Fetching Events from an Archive Node
+
+Since emitted event data is only stored in transaction data, events will only be persisted by an Archive Node. To fetch events from an Archive Node, you can use the `archive` property on the `Mina.Network` function. For example:
```

Comment:
Would suggest rewording the first sentence:

> Events are contained within zkApp transactions and persisted by Mina archive nodes.

Would reword the 2nd sentence similar to my suggestions in the earlier section.